### PR TITLE
Remove wide mode (4/4)

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
Markdown renderer

### Description
This PR removes the wide mode. It was supposed to make the document more readable, but imho the lightbox is more useful.

### Steps

- [x] Added implementation
- [x] Extended changelog
- [x] I read the [contribution documentation](https://github.com/hedgedoc/react-client/blob/main/CONTRIBUTING.md) and signed-off my commits to accept the DCO.

### CODE REVIEW

File: src/components/editor-page/editor-page.tsx [hunk 1]
```diff
@@ -8,7 +8,6 @@ import React, { Fragment, useCallback, useEffect, useRef, useState } from 'react
 import { useTranslation } from 'react-i18next'
 import { useSelector } from 'react-redux'
 import { useLocation } from 'react-router'
-import useMedia from 'use-media'
```
Comment (DerMolly) [inline src/components/editor-page/editor-page.tsx:11]: Do we use the package anywhere else? If not we should remove it.

**Review by DerMolly — CHANGES_REQUESTED** [review]

File: src/components/editor-page/editor-page.tsx (hunk 1 above)
Comment (mrdrogdrog) [reply src/components/editor-page/editor-page.tsx:11]: Good point. I removed it

**Review by DerMolly — APPROVED** [review]

File: src/components/editor-page/editor-pane/one-dark.scss [hunk 2]
```diff
@@ -1,11 +1,7 @@
...
- * SPDX-License-Identifier: MIT
+/*!
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
  *
- * Atom One Dark Theme
- * GitHub https://github.com/hikio/brackets-one-dark
+ * SPDX-License-Identifier: AGPL-3.0-only
  */
 
 /*
```
Comment (ErikMichelson) [inline src/components/editor-page/editor-pane/one-dark.scss:7]: Same thing as usual for this PR series...

**Review by ErikMichelson — COMMENTED** [review]
Works fine, as the images can be enlarged and video players have a full-screen button, I don't see any reason for the wide-view too.

File: src/components/editor-page/editor-pane/one-dark.scss (hunk 2 above)
Comment (mrdrogdrog) [reply src/components/editor-page/editor-pane/one-dark.scss:7]: Rebased

**Review by ErikMichelson — APPROVED** [review]

**Pull request merged**